add_tokens.py

```diff
@@ -0,0 +1,22 @@
+import pandas as pd
+
+def load_data():
+    try:
+        data = pd.read_csv('additiona_tokens.csv',index_col=[0])
```

Comment:
This relies on the script being run in a particular directory and in any case just statically returns a tiny dataframe which should just be a list. Make this a constant.

Reply:
The idea is that we will have a way to get user requests. CSV is just a temporary solution (or a proof-of-concept) and needs to be upgraded to a DB. Then we will have scripts ingest data from user requests.